modules/virt-creating-single-pvc-template-storage-pool.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-single-pvc-template-storage-pool_{context}"]
+= Creating a single `PVCTemplate` storage pool in a host path provisioner custom resource
+
+Specifying a single `pvcTemplate` (a standard Kubernetes `PersistentVolumeClaimSpec`)  allows you to specify a persistent volume claim (PVC) template, which is mounted on the node at the where you want to create hostpath provisioner (HPP) volumes. Using single `pvcTemplate`s can improve I/O performance. By configuring a cluster where each node has an additional disk to store the HPP, you effectively distribute many large volumes over multiple smaller virtual volumes.
+
+.Procedure
+
+. Ensure that your nodes have an available blank disk or partition that can be discovered by the `LocalStorage` Operator (LSO).
+
+. Install the `LocalStorage` Operator (LSO).
+//IS THERE A PLACE THIS IS DOCUMENTED? I COULD NOT FIND IT IN OCP. KUBEVIRT? IF SO WE CAN ADD IT TO THE ADDITIONAL RESOURCE XREFS.
+
+. Create a YAML file for the HPP CSI driver provisioner custom resource specifying a single `pvcTemplate`. For example:
++
+[source,terminal]
+----
+$ touch hostpathprovisioner_cr_singlepvc.yaml
+----
+
+. Edit the file. For example:
++
+[source,yaml]
+----
+apiVersion: hostpathprovisioner.kubevirt.io/v1beta1
+kind: HostPathProvisioner
+metadata:
+ name: hostpath-provisioner
+spec:
+ imagePullPolicy: IfNotPresent
+ storagePools:
+   - name: local
+  pvcTemplate:
+       volumeMode: Filesystem <1>
```

Comment:
Here you set Filesystem volume mode but in the multi storage pool you do block. And in both cases you use LSO, we should be consistent. The same comment applies about technically not needing LSO.

Reply:
Fixed.